pkg/lib/cockpit.d.ts

```diff
@@ -16,6 +16,8 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
+import '_internal/common'; // side-effecting import (`window` augmentations)
```

Comment:
Isn't *that* a good case for `import type`? You don't import any names, and this shouldn't land in the bundle unless actually necessary. It's really just a lint-time thing, after all?

Reply:
Yes.

Reply:
No.  Apparently you can't `import type 'x';`, only `import type { y } from 'x';`.

And anyone who includes `cockpit.js` is going to end up having a hard depend on that code anyway....